src/network/webtask.h

```diff
@@ -148,7 +148,7 @@ class WebTask : public NetworkTask {
 
     enum class State {
         // Initial state
-        Idle,
+        Initial,
```

Comment:
IMO the renaming only makes the state machine more difficult to understand, but I'm not gonna start bikeshedding over a name.